include/boost/gil/image_processing/numeric.hpp

```diff
@@ -136,21 +136,26 @@ inline auto generate_gaussian_kernel(std::size_t side_length, double sigma)
         throw std::invalid_argument("kernel dimensions should be odd and equal");
 
     const double denominator = 2 * boost::gil::detail::pi * sigma * sigma;
-    auto middle = side_length / 2;
+    const auto middle = side_length / 2;
```

Comment:
Nitpick: [Join the East Const Revolution!](https://hackingcpp.com/cpp/design/east_vs_west_const.html) ;-)